test/check-application

```diff
@@ -2035,9 +2048,12 @@ class TestApplication(testlib.MachineCase):
                        ' \u2192 8001/tcp')
         b.wait_not_in_text(f'#containers-containers tr:contains("{IMG_BUSYBOX}") dt:contains("Ports") + dd',
                            '7001/tcp')
+        b.wait_in_text(f'#containers-containers tr:contains("{IMG_BUSYBOX}") dt:contains("Ports") + dd',
+                       '0.0.0.0:4001-4004 \u2192 4001-4004/tcp')
```

Comment:
`\u2192` should be `→`

Reply:
I'm not actually sure what's going wrong here. I mirrored the tests above this one, and they use the same escape code/syntax

Reply:
I've build a VM locally, I'll test and help you figure it out

Reply:
![image](https://github.com/user-attachments/assets/e175c82e-c201-42b7-b3b0-8147c16e49bf)

So there is no 4001-4004 range here

Reply:
Oh my gosh, thank you! It's clear now in the first log message you sent 😅